Formula/netcat6.rb

```diff
@@ -2,7 +2,7 @@ class Netcat6 < Formula
   desc "Rewrite of netcat that supports IPv6, plus other improvements"
   homepage "https://www.deepspace6.net/projects/netcat6.html"
   url "https://mirrors.ocf.berkeley.edu/debian/pool/main/n/nc6/nc6_1.0.orig.tar.gz"
```

Comment:
Despite my comment above, I'm not sure if having a `url` stanza point to `deb.debian.org` is valid/good practice, even though the audit complains?

Reply:
Yes we can have a `url` point to that, and if the mirror becomes redundant then we remove it.